minecraft-modpack-gtnh/minecraft-modpack-gtnh.json

```diff
@@ -0,0 +1,102 @@
+{
+  "name": "minecraft-modpack-gtnh",
+  "display": "Minecraft Modpack - GT New Horizons",
+  "type": "minecraft-modpack",
```

Comment:
Yes, please use minecraft-java here.
In v3, this will help show some nicer icons on things, and so using modpack would change that.